### PR TITLE
✨ Fetch API and get Project by id 

### LINKED ISSUE #51: Fetch API
#### Description
betterplace.org API : https://github.com/betterplace/betterplace_apidocs

- [ ] Fetch betterplace.org API and get project by id
- [ ] Test get project by id

### PR DESCRIPTION
**Pull request by dhnazarenko**
fixes #51

- ✨ Fetch API
- ✨ Search project by id

### CODE REVIEW

File: src/server/betterPlaceDB.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  if (query) {
+    url += `query=${query}&`;
+  }
+
+  const response = await fetch(url);
+  if (!response.ok) {
+    const errorResult: ErrorResult = await response.json();
+    throw {
+      message: errorResult.status_message,
+      theMovieDBCode: errorResult.status_code,
```
Comment (Dev-dfm) [inline src/server/betterPlaceDB.ts]:
```suggestion
      betterPlaceDBCode: errorResult.status_code,
```

**Review by Dev-dfm — COMMENTED** [review]
lgtm 🚀

File: src/server/betterPlaceDB.ts (hunk 1 above)
Comment (dhnazarenko) [reply src/server/betterPlaceDB.ts]: Thanks!

**Review by lajanzen — APPROVED** [review]
🚀

**Pull request merged**